L.Map.Sync.js

```diff
@@ -46,6 +46,15 @@
             return this;
         },
 
+        // Checks if the maps is synced with anything
+        isSynced: function () {
+            if (this._syncMaps && Object.keys(this._syncMaps).length > 0) {
```

Comment:
could be simplified to `return this._syncMaps && Object.keys(this._syncMaps).length > 0`